tests/api_test.js

```diff
@@ -0,0 +1,69 @@
+var server = require("../main"),
+    chai = require("chai"),
+    chaiHTTP = require("chai-http"),
+    should = chai.should()
+
+chai.use(chaiHTTP)
+
+describe("Test data api", function () {
+
+    describe("GET to /api/v1.0/hypothesis/22f82a83-7b1b-4556-aeea-5b3b16a9b287", () => {
```

Comment:
replace hypothesis with probably data or hypothesis-results

Reply:
I will have to change the endpoints in data-api